include/conversation.php

```diff
@@ -144,222 +144,106 @@ function localize_item(&$item)
+			if (!$verb) {
+				return;
+			}
+			if ($item['object-type']=="" || $item['object-type']!== Activity\ObjectType::PERSON) {
+				return;
 			}
-		}
-
-		$A = '[url=' . Contact::magicLink($Alink) . ']' . $Aname . '[/url]';
-		$B = '[url=' . Contact::magicLink($Blink) . ']' . $Bname . '[/url]';
-		if ($Bphoto != "") {
-			$Bphoto = '[url=' . Contact::magicLink($Blink) . '][img]' . $Bphoto . '[/img][/url]';
-		}
-
-		$item['body'] = DI::l10n()->t('%1$s is now friends with %2$s', $A, $B)."\n\n\n".$Bphoto;
-
-	}
-	if (stristr($item['verb'], Activity::POKE)) {
-		$verb = urldecode(substr($item['verb'],strpos($item['verb'],'#')+1));
-		if (!$verb) {
-			return;
-		}
-		if ($item['object-type']=="" || $item['object-type']!== Activity\ObjectType::PERSON) {
-			return;
-		}
 
-		$Aname = $item['author-name'];
-		$Alink = $item['author-link'];
+			$Aname = $item['author-name'];
+			$Alink = $item['author-link'];
 
-		$xmlhead = "<" . "?xml version='1.0' encoding='UTF-8' ?" . ">";
+			$xmlhead = "<" . "?xml version='1.0' encoding='UTF-8' ?" . ">";
 
-		$obj = XML::parseString($xmlhead.$item['object']);
+			$obj = XML::parseString($xmlhead.$item['object']);
 
-		$Bname = $obj->title;
-		$Blink = $obj->id;
-		$Bphoto = "";
+			$Bname = $obj->title;
+			$Blink = $obj->id;
+			$Bphoto = "";
 
-		foreach ($obj->link as $l) {
-			$atts = $l->attributes();
-			switch ($atts['rel']) {
-				case "alternate": $Blink = $atts['href'];
-				case "photo": $Bphoto = $atts['href'];
+			foreach ($obj->link as $l) {
+				$atts = $l->attributes();
+				switch ($atts['rel']) {
+					case "alternate": $Blink = $atts['href'];
+					case "photo": $Bphoto = $atts['href'];
+				}
 			}
-		}
-
-		$A = '[url=' . Contact::magicLink($Alink) . ']' . $Aname . '[/url]';
-		$B = '[url=' . Contact::magicLink($Blink) . ']' . $Bname . '[/url]';
-		if ($Bphoto != "") {
-			$Bphoto = '[url=' . Contact::magicLink($Blink) . '][img=80x80]' . $Bphoto . '[/img][/url]';
-		}
 
-		/*
-		 * we can't have a translation string with three positions but no distinguishable text
-		 * So here is the translate string.
-		 */
-		$txt = DI::l10n()->t('%1$s poked %2$s');
-
-		// now translate the verb
-		$poked_t = trim(sprintf($txt, "", ""));
-		$txt = str_replace($poked_t, DI::l10n()->t($verb), $txt);
+			$A = '[url=' . Contact::magicLink($Alink) . ']' . $Aname . '[/url]';
+			$B = '[url=' . Contact::magicLink($Blink) . ']' . $Bname . '[/url]';
+			if ($Bphoto != "") {
+				$Bphoto = '[url=' . Contact::magicLink($Blink) . '][img=80x80]' . $Bphoto . '[/img][/url]';
+			}
 
-		// then do the sprintf on the translation string
+			/*
+			 * we can't have a translation string with three positions but no distinguishable text
+			 * So here is the translate string.
+			 */
+			$txt = DI::l10n()->t('%1$s poked %2$s');
 
-		$item['body'] = sprintf($txt, $A, $B). "\n\n\n" . $Bphoto;
+			// now translate the verb
+			$poked_t = trim(sprintf($txt, "", ""));
```

Comment:
Code standards: Please use single quotes unless you need string interpolation or escape characters.